### ISSUE TITLE
finetuning loss

### ISSUE THREAD

**Issue by whjxnyzh** [facebook/C3D#8]
Hi, I run the finetune example using ucf101 dataset. The loss look like this
```
I0528 21:46:49.915746 11590 solver.cpp:237] Iteration 20, lr = 0.0001
I0528 21:46:49.916153 11590 solver.cpp:87] Iteration 20, loss = 4.90706
I0528 21:47:27.747942 11590 solver.cpp:237] Iteration 40, lr = 0.0001
I0528 21:47:27.748319 11590 solver.cpp:87] Iteration 40, loss = 4.57005
I0528 21:48:05.530530 11590 solver.cpp:237] Iteration 60, lr = 0.0001
I0528 21:48:05.530972 11590 solver.cpp:87] Iteration 60, loss = 5.05493
I0528 21:48:43.342098 11590 solver.cpp:237] Iteration 80, lr = 0.0001
I0528 21:48:43.342476 11590 solver.cpp:87] Iteration 80, loss = 4.09023
I0528 21:49:21.167407 11590 solver.cpp:237] Iteration 100, lr = 0.0001
I0528 21:49:21.167830 11590 solver.cpp:87] Iteration 100, loss = 4.71882
I0528 21:49:58.997171 11590 solver.cpp:237] Iteration 120, lr = 0.0001
I0528 21:49:58.997553 11590 solver.cpp:87] Iteration 120, loss = 3.3148
I0528 21:50:36.777606 11590 solver.cpp:237] Iteration 140, lr = 0.0001
I0528 21:50:36.777989 11590 solver.cpp:87] Iteration 140, loss = 2.39711
I0528 21:51:14.583552 11590 solver.cpp:237] Iteration 160, lr = 0.0001
I0528 21:51:14.584013 11590 solver.cpp:87] Iteration 160, loss = 3.69446
I0528 21:51:52.391942 11590 solver.cpp:237] Iteration 180, lr = 0.0001
I0528 21:51:52.392320 11590 solver.cpp:87] Iteration 180, loss = 3.81515
I0528 21:52:30.216404 11590 solver.cpp:237] Iteration 200, lr = 0.0001
I0528 21:52:30.216820 11590 solver.cpp:87] Iteration 200, loss = 2.30991
I0528 21:53:08.017086 11590 solver.cpp:237] Iteration 220, lr = 0.0001
I0528 21:53:08.017467 11590 solver.cpp:87] Iteration 220, loss = 4.23676
I0528 21:53:45.828099 11590 solver.cpp:237] Iteration 240, lr = 0.0001
I0528 21:53:45.828479 11590 solver.cpp:87] Iteration 240, loss = 3.21753
I0528 21:54:23.647207 11590 solver.cpp:237] Iteration 260, lr = 0.0001
I0528 21:54:23.647586 11590 solver.cpp:87] Iteration 260, loss = 2.07026
I0528 21:55:01.485985 11590 solver.cpp:237] Iteration 280, lr = 0.0001
I0528 21:55:01.486362 11590 solver.cpp:87] Iteration 280, loss = 1.96745
I0528 21:55:39.291198 11590 solver.cpp:237] Iteration 300, lr = 0.0001
I0528 21:55:39.291574 11590 solver.cpp:87] Iteration 300, loss = 1.99167
I0528 21:56:17.111016 11590 solver.cpp:237] Iteration 320, lr = 0.0001
I0528 21:56:17.111387 11590 solver.cpp:87] Iteration 320, loss = 1.50536
I0528 21:56:54.930027 11590 solver.cpp:237] Iteration 340, lr = 0.0001
I0528 21:56:54.930405 11590 solver.cpp:87] Iteration 340, loss = 3.28589
I0528 21:57:32.727215 11590 solver.cpp:237] Iteration 360, lr = 0.0001
I0528 21:57:32.727594 11590 solver.cpp:87] Iteration 360, loss = 2.74588
I0528 21:58:10.523322 11590 solver.cpp:237] Iteration 380, lr = 0.0001
I0528 21:58:10.523720 11590 solver.cpp:87] Iteration 380, loss = 0.946066
I0528 21:58:48.311733 11590 solver.cpp:237] Iteration 400, lr = 0.0001
I0528 21:58:48.312110 11590 solver.cpp:87] Iteration 400, loss = 2.00241
I0528 21:59:26.104333 11590 solver.cpp:237] Iteration 420, lr = 0.0001
I0528 21:59:26.104742 11590 solver.cpp:87] Iteration 420, loss = 2.41513
I0528 22:00:03.892549 11590 solver.cpp:237] Iteration 440, lr = 0.0001
I0528 22:00:03.892981 11590 solver.cpp:87] Iteration 440, loss = 1.25305
I0528 22:00:41.703632 11590 solver.cpp:237] Iteration 460, lr = 0.0001
I0528 22:00:41.704085 11590 solver.cpp:87] Iteration 460, loss = 1.69272
I0528 22:01:19.531800 11590 solver.cpp:237] Iteration 480, lr = 0.0001
I0528 22:01:19.532176 11590 solver.cpp:87] Iteration 480, loss = 2.35864
I0528 22:01:57.354773 11590 solver.cpp:237] Iteration 500, lr = 0.0001
I0528 22:01:57.355154 11590 solver.cpp:87] Iteration 500, loss = 3.20118
```
My GPU is K20, I change the batch size from 30 to 10
is this loss normal?

**Comment by dutran** [comment]
Try to reduce lr by 3 (as you reduce batch size by a factor of 3). This looks good. Random loss is ln(101) ~ 4.62.

**Comment by whjxnyzh** [comment]
Thank you very much @dutran I will try it out

**Comment by whjxnyzh** [comment]
Great, the loss is very small now
```
I0529 08:51:22.795929  7108 solver.cpp:237] Iteration 20020, lr = 0.0001
I0529 08:51:22.796392  7108 solver.cpp:87] Iteration 20020, loss = 0.00696523
I0529 08:52:00.631763  7108 solver.cpp:237] Iteration 20040, lr = 0.0001
I0529 08:52:00.632230  7108 solver.cpp:87] Iteration 20040, loss = 3.70235e-05
I0529 08:52:38.457860  7108 solver.cpp:237] Iteration 20060, lr = 0.0001
I0529 08:52:38.458324  7108 solver.cpp:87] Iteration 20060, loss = 0.0188635
I0529 08:53:16.247462  7108 solver.cpp:237] Iteration 20080, lr = 0.0001
I0529 08:53:16.247931  7108 solver.cpp:87] Iteration 20080, loss = 0.0098978
I0529 08:53:54.076977  7108 solver.cpp:237] Iteration 20100, lr = 0.0001
I0529 08:53:54.077445  7108 solver.cpp:87] Iteration 20100, loss = 0.039086
I0529 08:54:31.882534  7108 solver.cpp:237] Iteration 20120, lr = 0.0001
I0529 08:54:31.883000  7108 solver.cpp:87] Iteration 20120, loss = 0.0021823
I0529 08:55:09.673733  7108 solver.cpp:237] Iteration 20140, lr = 0.0001
I0529 08:55:09.674206  7108 solver.cpp:87] Iteration 20140, loss = 0.0851032
I0529 08:55:47.512164  7108 solver.cpp:237] Iteration 20160, lr = 0.0001
I0529 08:55:47.512662  7108 solver.cpp:87] Iteration 20160, loss = 0.00216188
I0529 08:56:25.341374  7108 solver.cpp:237] Iteration 20180, lr = 0.0001
I0529 08:56:25.341886  7108 solver.cpp:87] Iteration 20180, loss = 0.00296394
I0529 08:57:03.134181  7108 solver.cpp:237] Iteration 20200, lr = 0.0001
I0529 08:57:03.134677  7108 solver.cpp:87] Iteration 20200, loss = 0.0216006
I0529 08:57:40.966392  7108 solver.cpp:237] Iteration 20220, lr = 0.0001
I0529 08:57:40.966887  7108 solver.cpp:87] Iteration 20220, loss = 0.00295561
I0529 08:58:18.788378  7108 solver.cpp:237] Iteration 20240, lr = 0.0001
I0529 08:58:18.788894  7108 solver.cpp:87] Iteration 20240, loss = 0.0404173
I0529 08:58:56.575695  7108 solver.cpp:237] Iteration 20260, lr = 0.0001
I0529 08:58:56.576161  7108 solver.cpp:87] Iteration 20260, loss = 0.0247827
I0529 08:59:34.381561  7108 solver.cpp:237] Iteration 20280, lr = 0.0001
I0529 08:59:34.382026  7108 solver.cpp:87] Iteration 20280, loss = 0.00695413
I0529 09:00:12.204666  7108 solver.cpp:237] Iteration 20300, lr = 0.0001
I0529 09:00:12.205132  7108 solver.cpp:87] Iteration 20300, loss = 0.03263
I0529 09:00:49.999734  7108 solver.cpp:237] Iteration 20320, lr = 0.0001
I0529 09:00:50.000200  7108 solver.cpp:87] Iteration 20320, loss = 0.0037585
I0529 09:01:27.809442  7108 solver.cpp:237] Iteration 20340, lr = 0.0001
I0529 09:01:27.809955  7108 solver.cpp:87] Iteration 20340, loss = 0.00100278
I0529 09:02:05.637639  7108 solver.cpp:237] Iteration 20360, lr = 0.0001
I0529 09:02:05.638101  7108 solver.cpp:87] Iteration 20360, loss = 0.0324224
```

**Comment by dutran** [comment]
I'm glad to hear, good luck with your experiments.

**Comment by sainisanjay** [comment]
@whjxnyzh i am getting stable loss around (3.2867), can you send your  parameter (solver.prototxt) file.
